.circleci/config.yml

```diff
@@ -50,3 +50,39 @@ jobs: # a collection of steps
       - store_test_results: # Upload test results for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
           path: test_results
       # See https://circleci.com/docs/2.0/deployment-integrations/ for example deploy configs
+  deploy:
+      docker:
+        - image: circleci/python:2.7
```

Comment:
Oh neat, we are using python for the docker image!

Reply:
Did you have to download pythin to your local machine?